core/random.rbs

```diff
@@ -129,141 +58,53 @@ class Random < Object
-  #
-  # The argument *n* specifies the length of the result string.
-  #
-  # If *n* is not specified or is nil, 16 is assumed. It may be larger in future.
-  #
-  # The result may contain any byte: "x00" - "xff".
-  #
-  #     require 'securerandom'
-  #
-  #     SecureRandom.random_bytes #=> "\xD8\\\xE0\xF4\r\xB2\xFC*WM\xFF\x83\x18\xF45\xB6"
-  #     SecureRandom.random_bytes #=> "m\xDC\xFC/\a\x00Uf\xB2\xB2P\xBD\xFF6S\x97"
-  #
-  # If a secure random number generator is not available, `NotImplementedError` is
-  # raised.
-  #
-  def random_bytes: (?Integer? n) -> String
+  def initialize_copy: (self object) -> self
 
-  # Generates formatted random number from raw random bytes. See Random#rand.
-  #
-  def random_number: () -> Float
-                   | (?Float? n) -> Float
-                   | (?Integer? n) -> Integer
-                   | (?Numeric? n) -> Numeric
-                   | (?::Range[Float]? n) -> Float
-                   | (?::Range[Integer]? n) -> Integer
-                   | (?::Range[Numeric]? n) -> Numeric
+  def left: () -> Integer
 
-  # SecureRandom.urlsafe_base64 generates a random URL-safe base64 string.
-  #
-  # The argument *n* specifies the length, in bytes, of the random number to be
-  # generated. The length of the result string is about 4/3 of *n*.
-  #
-  # If *n* is not specified or is nil, 16 is assumed. It may be larger in the
-  # future.
-  #
-  # The boolean argument *padding* specifies the padding. If it is false or nil,
-  # padding is not generated. Otherwise padding is generated. By default, padding
-  # is not generated because "=" may be used as a URL delimiter.
-  #
-  # The result may contain A-Z, a-z, 0-9, "-" and "_". "=" is also used if
-  # *padding* is true.
-  #
-  #     require 'securerandom'
-  #
-  #     SecureRandom.urlsafe_base64 #=> "b4GOKm4pOYU_-BOXcrUGDg"
-  #     SecureRandom.urlsafe_base64 #=> "UZLdOkzop70Ddx-IJR0ABg"
-  #
-  #     SecureRandom.urlsafe_base64(nil, true) #=> "i0XQ-7gglIsHGV2_BNPrdQ=="
-  #     SecureRandom.urlsafe_base64(nil, true) #=> "-M8rLhr7JEpJlqFGUMmOxg=="
-  #
-  # If a secure random number generator is not available, `NotImplementedError` is
-  # raised.
-  #
-  # See RFC 3548 for the definition of URL-safe base64.
-  #
-  def urlsafe_base64: (?Integer? n, ?boolish padding) -> String
+  def marshal_dump: () -> Array[Integer]
 
-  # SecureRandom.uuid generates a random v4 UUID (Universally Unique IDentifier).
-  #
-  #     require 'securerandom'
-  #
-  #     SecureRandom.uuid #=> "2d931510-d99f-494a-8c67-87feb05e1594"
-  #     SecureRandom.uuid #=> "bad85eb9-0713-4da7-8d36-07a8e4b00eab"
-  #     SecureRandom.uuid #=> "62936e70-1815-439b-bf89-8492855a7e6b"
-  #
-  # The version 4 UUID is purely random (except the version). It doesn't contain
-  # meaningful information such as MAC addresses, timestamps, etc.
-  #
-  # The result contains 122 random bits (15.25 random bytes).
-  #
-  # See RFC 4122 for details of UUID.
-  #
-  def uuid: () -> String
```

Comment:
When I run `rbs prototype runtime --merge Random`, the `Random::Formatter` module is removed from the definition file.

Do I need to specify any options when I run `rbs prototype`?

Reply:
Removing `Random::Formatter` seems to be causing the test to fail...